website/templates/emails/pending_retraction_admin.txt.mako

```diff
@@ -0,0 +1,15 @@
+Hello ${user.fullname},
+
+
+I just wanted to let you know, ${initiated_by} has initiated a retraction for the following registration: ${registration_link}
```

Comment:
Throughout, we presumably want to use "(ET)" instead of "(EDT)" because we will not be observing daylight savings time year round.

http://en.wikipedia.org/wiki/Eastern_Time_Zone